src/index.js

```diff
@@ -42,46 +44,63 @@ export function convertSchemaToHtml(schema, scoped = false) {
   return html
 }
 
-export function buildParagraph(el) {
-  if (el?.children) {
-    return `<p>${convertSchemaToHtml(el?.children)}</p>`
+function getClass(tag, classes) {
+  if (classes && classes[tag]) {
+    return classes[tag]
+  } else {
+    return null
   }
 }
 
-export function buildHeading(el) {
-  if (el?.children) {
-    return `<h${el?.level}>${convertSchemaToHtml(el?.children)}</h${el?.level}>`
-  }
+function outputAttributes(attributes) {
+  if (!attributes && attributes?.class) return ''
+  return Object.keys(attributes)
+    .map(key => {
+      if (attributes[key]) {
+        return ` ${key}="${attributes[key]}"`
```

Comment:
Is this space needed here? Could it be added during the join?

I think you want to make the choice of adding a leading `<space>` explicit in the code. Right now, it's a subtle detail.